ui-tests/cypress/support/pageObjects/app/AdminPage.js

```diff
@@ -13,12 +14,16 @@ export default class AdminPage extends AbstractPage {
     super();
     this.menus   = new AdminMenus(this);
     this.content = new content(this);
+    this.dialog  = new AdminDialog();
   }
 
   get() {
     return this.parent.get()
                .children(htmlElements.body);
   }
+  getAdminDialog() {
```

Comment:
Use the same name as `AppPage` (i.e. _getDialog_), to make it transparent from the test point of view which type of page is used; you could even leave the method on the `AbstractPage` and put only the definition of what a _Dialog_ is in the extending classes